playbooks/bats_pipeline_katello_nightly.yml

```diff
@@ -12,7 +12,7 @@
     foreman_installer_disable_system_checks: true
     foreman_installer_options_internal_use_only:
       - "--foreman-initial-admin-password {{ foreman_installer_admin_password }}"
-      - "--katello-enable-ostree=true"
+      - "--katello-enable-ostree=false"
```

Comment:
This parameter no longer exists. I wonder if this playbook is still used.

Reply:
*probably* not.

`centos7-bats` box refers to `playbooks/bats_pipeline_nightly.yml` which doesn't exist, and `centos7-katello-bats-ci` (in ` 99-local.yaml.example`) refers to this one, but I don't think anyone uses those boxes today?

Reply:
Let keep this PR to remove the ostree parameter and I will create another PR to remove that playbook if required.